src/PKSim.Matlab/ApplicationStartup.cs

```diff
@@ -14,36 +14,23 @@ internal class ApplicationStartup
    {
       private static bool _initialized;
 
-      public static void Initialize(string dimensionFilePath = null, string databaseFilePath = null, string pkParameterFilePath = null)
+      public static void Initialize()
       {
          if (_initialized) return;
 
          redirectNHibernateAssembly();
 
-         new ApplicationStartup().initializeForMatlab(dimensionFilePath, databaseFilePath, pkParameterFilePath);
+         new ApplicationStartup().initializeForMatlab();
          _initialized = true;
       }
 
-      private void initializeForMatlab(string dimensionFilePath, string databaseFilePath, string pkParameterFilePath)
+      private void initializeForMatlab()
       {
          if (IoC.Container != null)
             return;
 
          InfrastructureRegister.Initialize();
          var container = IoC.Container;
-         var pksimConfiguration = container.Resolve<IPKSimConfiguration>();
-
-         //path was specified, update the default path
-         if (!string.IsNullOrEmpty(dimensionFilePath))
-            pksimConfiguration.DimensionFilePath = dimensionFilePath;
```

Comment:
We do not need to set those guys in tests now. They can rely on the portable logic so to speak